src/app/components/Search/index.tsx

```diff
@@ -28,6 +28,11 @@ const SearchForm = styled('form', {
   // Approximate size of TextField without helperText, so helperText floats.
   height: '47px',
 
+  // Show helper text when focused or filled.
+  ':not(:hover, :focus-within) :not(.MuiFormHelperText-filled).MuiFormHelperText-root': {
```

Comment:
This is so hard to maintain, even comment is saying show when css selector is actually hiding element. Can't we just render SearchSuggestions when needed? Is that a case we are trying to accomplish?

```
 helperText={
          (isHovered || isFocused) && (
            <SearchSuggestions
              onClickSuggestion={suggestion => {
                setValue(suggestion)
              }}
            />
          )
        }
``` ?

Reply:
Yeah :/

I'm not happy about making it visible on hover, but without it: focus empty, see suggestions, click suggestions, suggestions disappear and don't work

Reply:
I'll simplify to `value && <SearchSuggestions` and revisit later :D